instrumentation/ratpack/ratpack-1.7/library/src/main/java/io/opentelemetry/instrumentation/ratpack/v1_7/RatpackTelemetryBuilder.java

```diff
@@ -173,6 +174,13 @@ public RatpackTelemetryBuilder setServerSpanNameExtractor(
     return this;
   }
 
+  @CanIgnoreReturnValue
+  public RatpackTelemetryBuilder configure(CommonConfig config) {
+    clientBuilder.configure(config);
+    serverBuilder.configure(config);
+    return this;
+  }
```

Comment:
I think this method should not be a part of the public api. Other instrumentations use a class in internal package to share code between the agent and library instrumentation. For example see https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/java-http-client/library/src/main/java/io/opentelemetry/instrumentation/httpclient/internal/JavaHttpClientInstrumenterBuilderFactory.java